hdate/__init__.py

```diff
@@ -402,9 +396,6 @@ def get_reading(self, diaspora):
             return 0
         elif self._h_weeks == 4:
             if self._h_new_year_weekday == 7:
-                # Simhat tora in israel
```

Comment:
why did you deleted this?

Reply:
Impossible to reach this case.
Unittests cover all possible years. This case is never reached.